### PR TITLE
set default values for metrics and network mapper to solve reuse-values

### PR DESCRIPTION
**Pull request by Moran-k**

### CODE REVIEW

File: charts/k8s-watcher/templates/network-mapper/cluster-role.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/cluster-role.yaml:1]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

File: charts/k8s-watcher/templates/network-mapper/configmap.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/configmap.yaml:1]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

File: charts/k8s-watcher/templates/network-mapper/daemonset.yaml [hunk 3]
```diff
@@ -1,17 +1,15 @@
 {{- define "network_mapper.daemonset.container" }}
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/daemonset.yaml:2]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

File: charts/k8s-watcher/templates/network-mapper/deployment.yaml [hunk 4]
```diff
@@ -1,15 +1,13 @@
 {{- define "network_mapper.deployment.container" }}
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/deployment.yaml:2]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

File: charts/k8s-watcher/templates/network-mapper/role.yaml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/role.yaml:1]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

File: charts/k8s-watcher/templates/network-mapper/roles-binding.yaml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.serviceAccount.create -}}
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/roles-binding.yaml:2]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

File: charts/k8s-watcher/templates/network-mapper/service.yaml [hunk 7]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.network_mapper ((.Values.watcher.networkMapper).enable | default false) }}
+{{- if (.Values.watcher.networkMapper).enable | default false }}
```
Comment (nirbenator) [inline charts/k8s-watcher/templates/network-mapper/service.yaml:1]:
```suggestion
{{- if (.Values.watcher.networkMapper).enable | default true }}
```

**Review by nirbenator — CHANGES_REQUESTED** [review]
1. we want to enable mapper by default
2. the chart is yuck. i also don't like the version. we'll handle it next sprint

**Comment by Moran-k** [comment]
CU-860rk698r

**Review by nirbenator — APPROVED** [review]

**Pull request merged**